libs/server/Objects/Types/GarnetObjectStoreOutput.cs

```diff
@@ -1,24 +1,52 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+using System;
 using Tsavorite.core;
 
 namespace Garnet.server
 {
+    /// <summary>
+    /// Flags for object store outputs.
+    /// </summary>
+    [Flags]
+    public enum ObjectStoreOutputFlags : byte
+    {
+        /// <summary>
+        /// No flags set
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        /// Remove key
+        /// </summary>
+        RemoveKey = 1,
+
+        /// <summary>
+        /// Wrong type of object
+        /// </summary>
+        WrongType = 1 << 1,
+    }
+
     /// <summary>
     /// Output type used by Garnet object store.
     /// </summary>
     public struct GarnetObjectStoreOutput
     {
         /// <summary>
-        /// span byte and memory
+        /// Span byte and memory
```

Comment:
Add a comment to GOSO that we should only add fields that need to flow from backend (IFunctions) to frontend (caller of GarnetApi) in that specific direction.